src/preferences/dialog/dlgprefwaveformdlg.ui

```diff
@@ -330,6 +343,16 @@
        </property>
       </widget>
      </item>
+     <item row="12" column="1">
+      <widget class="QCheckBox" name="beatGridLinesCheckBox">
+       <property name="toolTip">
+        <string>Display beat grid lines on waveform</string>
```

Comment:
@Be-ing  I've resolved the issue with the preference being ignored when starting Mixxx. Which lines am I changing to "Beat Grid" and "Show Beat Markers"? I'll push all the changes together.